client/landing/stepper/declarative-flow/internals/components/step-route/index.tsx

```diff
@@ -1,5 +1,5 @@
 import classnames from 'classnames';
-import { useEffect } from 'react';
+import { PropsWithChildren, useEffect } from 'react';
```

Comment:
Should this not be a type import?
```suggestion
import { type PropsWithChildren, useEffect } from 'react';
```